openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <table>
+    <tr>
+      <th>
+        <select class="form-control" v-model="selectedIdentifier" name="name">
+          <option disabled value="">Select one</option>
+          <option v-for="idConfig in identifierConfigsByKey" :key="idConfig.name" :value="idConfig.name">
+            {{idConfig.label}}
+          </option>
+        </select>
+      </th>
+      <th>
+        <input class="form-control" type="text" name="value" id="id-value" v-model="inputValue" @keyup.enter=setIdentifier>
+      </th>
+      <th>
+        <button class="form-control" type="button" name="set" :disabled="!setButtonEnabled" @click=setIdentifier>Set</button>
+      </th>
+    </tr>
+    <tr :key="identifier.name" v-for="identifier in assignedIdentifiersWithConfigs">
```

Comment:
And here we can do:
```suggestion
    <tr v-for="(value, name) in assignedIdentifiers" :key="name">
```

and then call `identifiersConfigsByKey[name].label`